homeassistant/components/modbus/__init__.py

```diff
@@ -285,6 +286,7 @@
             vol.Optional(CONF_STATE_CLASS): SENSOR_STATE_CLASSES_SCHEMA,
             vol.Optional(CONF_UNIT_OF_MEASUREMENT): cv.string,
             vol.Optional(CONF_SLAVE_COUNT, default=0): cv.positive_int,
+            vol.Optional(CONF_MIN_VALUE, default=0): number_validator,
```

Comment:
What does a min. value mean ? the modbus integration should not modify the values, but deliver what we get from the modbus device.

Apart from that I would have expected a max. value as well.

Reply:
Its abs(val) so the min value is the smallest abs value

Reply:
I have a modbus device that measures current.
It has noise around zero (it is not reliable for currents below 30mA).
It has onboard registers for min value suppression but they dont work very well, so makes more sense to have it here where they are easily accessible (and work !).

Reply:
Ok, but what about this....mi fridge measures temperature and it has noise around zero, but are normally -6 degrees.

As I wrote I am not against a min, but it makes little sense in the general case without also providing a max.

Reply:
Do you have a use case for a max (except syntactic completeness)?
If you can think of one I will add one ... I did not want to add code for which there was no use case, and I could not think of one

Reply:
Yes my voltage sensor...it is accurate between 0 and 250, above that it becomes random numbers.

Second use case, my fridge which would have max_value=0 (since it must deliver negative values).